styles.css

```diff
@@ -81,8 +88,52 @@ header {
 	text-transform: uppercase;
 }
 
+main {
```

Comment:
I would suggest setting a maximum width to the section so that you have more control over how it appears.